README.md

```diff
@@ -8,7 +8,7 @@
 
 # Visual Studio Team Services Extension for Google Play
 
-![Build Status](https://mseng.visualstudio.com/_apis/public/build/definitions/b924d696-3eae-4116-8443-9a18392d8544/5350/badge?branch=master)
+[![Build status](https://dev.azure.com/mseng/AzureDevOps/_apis/build/status/CrossPlatform.google-play-vsts-extension.GitHub.CI)](https://dev.azure.com/mseng/AzureDevOps/_build/latest?definitionId=5350)
```

Comment:
Two links?

Reply:
This is the markdown copied from the build pipeline options page. It will link to the build definition instead of showing an image when clicked.